spec/mailers/subscription_mailer_spec.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+describe SubscriptionMailer do
+  fab!(:user) { Fabricate(:user) }
+
+  subject { SubscriptionMailer.confirm_unsubscribe(user) }
+
+  it "contains the right URL" do
+    expect(subject.body).to include("http://test.localhost/email/unsubscribe/#{UnsubscribeKey.last.key}")
```

Comment:
Could this be something like this instead?

```suggestion
    expect(subject.body).to include("#{Discourse_base_url}/email/unsubscribe/#{UnsubscribeKey.last.key}")
```